lib/time.ts

```diff
@@ -0,0 +1,24 @@
+export const millisecond = 1000
+
+export const second = millisecond * 60
+
+export const minute = second * 60
+
+/**
+ * Converts milliseconds into the 'HH:MM' format
```

Comment:
I've said `HH:MM` in a few places but it's actually `MM:SS`. Note to self to fix that

Reply:
You would actually be right if you use HH:MM, because that's what you are going to measure if you use `millisecond = 1000` and `second = millisecond * 60`, because it should be `millisecond = 1` and `second = 1000` (one second is clearly **not** 60 milliseconds 😉)
We don't _need_ a label for millisecond, because it's the unit of measure in js

Reply:
Hahaha! Yeah, i was browsing the test and saw that they all are based on this labels, so all tests pass because they are not testing with REAL time, only with the labels of time 🤣